src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const numOne = Big(numberOne);
+  const numTwo = Big(numberTwo);
+
+  switch (operation) {
+    case '/':
+      if (numTwo === 0) {
+        return 'undefined';
+      }
+      return numOne.div(numTwo).toString();
+    case 'X':
+      return numOne.times(numTwo).toString();
+    case '-':
+      return numOne.minus(numTwo).toString();
+    case '+':
+      return numOne.plus(numTwo).toString();
+    default:
+      return 'error';
+  }
+};
```

Comment:
You are missing the operation for the modulus operator, please look into that and make change

Reply:
I think I don't use modulus operation for this calculator. Should I? Or do you refer to the division operator? I'm using "/“ instead of "÷"

Reply:
I did change the symbol now :)